packages/itwinui-react/src/core/LabeledInput/LabeledInput.test.tsx

```diff
@@ -28,20 +40,22 @@ it('should render disabled component', () => {
     <LabeledInput label='some label' disabled />,
   );
   assertBaseElement(container);
-  expect(container.querySelector('.iui-input-flex-container')).toBeTruthy();
   const label = container.querySelector('.iui-input-label');
   expect(label).toBeTruthy();
   expect(label).toHaveAttribute('data-iui-disabled', 'true');
   getByText('some label');
-  expect((container.querySelector('input') as HTMLInputElement).disabled).toBe(
-    true,
-  );
+  expect(
+    (
+      container.querySelector(
+        '.iui-input-with-icon > input',
+      ) as HTMLInputElement
+    ).disabled,
+  ).toBe(true);
 });
 
 it('should handle required attribute', () => {
   const { container } = render(<LabeledInput label='some label' required />);
   assertBaseElement(container);
-  expect(container.querySelector('.iui-input-flex-container')).toBeTruthy();
```

Comment:
Removed some `expect(inputContainer).toBeTruthy()` test like this since this is already tested in `assertBaseElement`.